FreeRTOS-Plus/Demo/corePKCS11_MQTT_Mutual_Auth_Windows_Simulator/corePKCS11_MQTT_Mutual_Auth.vcxproj.filters

```diff
@@ -72,6 +72,7 @@
     <ClCompile Include="PKCS11_Mqtt_MutualAuthDemo.c">
       <Filter>Source</Filter>
     </ClCompile>
+    <ClCompile Include="..\..\Source\Application-Protocols\network_transport\mbedtls_rng_pkcs11.c" />
```

Comment:
Is there a reason this does not have a filter associated with it?

Reply:
I honestly have no idea, this is a file generated by Visual Studio, I didn't personally do anything different here